src/Native/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -238,7 +238,17 @@ C_FUNC(\Name):
 #define THREAD__VALIDATEEXINFOPOP C_FUNC(_ZN6Thread17ValidateExInfoPopEP6ExInfoPv)
 
 .macro INLINE_GETTHREAD
-        call    C_FUNC(RhpGetThread)
+        // Inlined version of call C_FUNC(RhpGetThread)
+       .att_syntax
```

Comment:
It was more than that. I also was not able to figure out how to rewrite `leaq tls_CurrentThread@TLSLD(%rip), %rdi` either.